### DISCUSSION TITLE
ubuntu 20.04 Battery Module Unable to Access Battery Information

### DISCUSSION THREAD

**Discussion by ddoherty03** [starship/starship#2089] [Q&A] [answered]
Using the standard configuration, I get the following message on my laptop after every prompt: 

    (starship::modules::battery) Unable to access battery information:
    Unable to calculate voltage

I would like to see the battery info in the prompt, so I don't want to disable it.

I can get battery status at the command prompt with

    $ cat /sys/class/power/BAT0

How is starship trying to query battery information that it is failing?

**Comment by andytom** [comment, marked as answer]
We use [rust-battery](https://github.com/svartalf/rust-battery) to get the battery information, this looks like an upstream issue with the library.
